this patch adds flag -fpermissive which is needed to build with GCC-4.7.x
and also adds a missing include
Pablo Escobar - sciCORE
--- contralign.orig/src/Makefile	2008-08-15 03:06:30.000000000 +0200
+++ contralign/src/Makefile	2014-02-04 17:25:31.364788280 +0100
@@ -23,7 +23,7 @@
 	-ffast-math -funroll-all-loops -funsafe-math-optimizations \
 	-fpeel-loops -Winline --param large-function-growth=100000 \
 	--param max-inline-insns-single=100000 \
-	--param inline-unit-growth=100000
+	--param inline-unit-growth=100000 -fpermissive
 
 OTHER_FLAGS = 
 
--- contralign.orig/src/Utilities.cpp	2008-08-14 02:08:30.000000000 +0200
+++ contralign/src/Utilities.cpp	2014-02-04 17:25:05.483971266 +0100
@@ -2,6 +2,7 @@
 // Utilities.cpp
 //////////////////////////////////////////////////////////////////////
 
+#include <limits.h>
 #include "Utilities.hpp"
 
 bool toggle_error = false;
